buddyboss/buddyboss-platform
# 584 - Fixed the issue with top level domain characters more than 5

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #584 .

### How to test the changes in this Pull Request:

1. Make sure that link preview is enabled
2. Go to the Activity feed
3. Post this link https://ldx.design/

### Proof Screenshots or Video
https://www.loom.com/share/38e88e52c3f1464ca8343f0d5d13a8b1



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
